runtime/browser/demo/demo.js

```diff
@@ -40,7 +40,7 @@ let stages = [{
 }];
 
 require('../lib/auto-tabs.js');
-require('../lib/x-toast.js');
```

Comment:
IIUC the vr-demo also uses x-toast:
https://github.com/PolymerLabs/arcs/blob/master/runtime/browser/vr-demo/vr-demo.js#L83
so this PR doesn't really change any of that. Let's discuss more tomorrow.